client/src/actions/auth.js

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
+import { setAlert } from './alert';
+import { REGISTER_COMPLETE, REGISTER_FAIL } from './types';
+
+// Signup user
+export const signUp =
+	({ userName, email, password }) =>
+	async dispatch => {
+		const config = {
+			headers: {
+				'Content-Type': 'application/json',
+			},
+		};
+		const body = JSON.stringify({ userName, email, password });
+		try {
+			const res = await axios.post('/app/register', body, config);
+
+			dispatch({
+				type: REGISTER_COMPLETE,
+				payload: res.data,
+			});
+		} catch (err) {
```

Comment:
What's the difference between err and error, are they having different value "?

Reply:
im catching the error in err and then saving it in errors

Reply:
Just catch the err and console.log(err)

Reply:
there are multiple errors

Reply:
i tried console.log(err) now all the errors wont show. thanks for your input.